Dockerfile

```diff
@@ -0,0 +1,38 @@
+# golang:1.16 image created 2021-06-24T00:31:06.02014601Z 
+FROM docker.io/library/golang@sha256:be99fa59acd78bb22a41bbc1e15ebfab2262498ee0c2e28c3d09bc44d51d1774 AS builder
+
+# Build the preflight binary
+COPY . /go/src/preflight
+WORKDIR /go/src/preflight
+RUN make build
+
+# Download and extract openscap 
+RUN curl -L https://github.com/OpenSCAP/openscap/releases/download/1.3.5/openscap-1.3.5.tar.gz | tar -xzv -C /tmp 
```

Comment:
It would be great to define the version in an ENV block. Make it easier to change the version in the case of the upgrade.

Reply:
This was something that I was going to do in a follow up PR but it makes sense to just add them now. I added ARG instructions for these, as they are only needed during the build stage and are not needed in the running image.